modules/lkqdBidAdapter.js

```diff
@@ -36,21 +36,32 @@ function buildRequests(validBidRequests, bidderRequest) {
   for (let i = 0; i < validBidRequests.length; i++) {
     let bidRequest = validBidRequests[i];
 
+    let sizes = [];
     // if width/height not provided to the ad unit for some reason then attempt request with default 640x480 size
-    if (!bidRequest.sizes || !bidRequest.sizes.length) {
+    if ((!bidRequest.sizes || !bidRequest.sizes.length) && (!bidRequest.mediaTypes || !bidRequest.mediaTypes.banner || !bidRequest.mediaTypes.banner.sizes || !bidRequest.mediaTypes.banner.sizes.length)) {
```

Comment:
Instead of checking each level in `bidRequest.mediaTypes.banner.sizes`, use `utils.deepAccess`

Reply:
Fixed